app/models/primer.rb

```diff
@@ -1,9 +1,9 @@
 class Primer < ActiveRecord::Base
 
   has_many :marker_assays_A, class_name: 'MarkerAssay',
```

Comment:
No no, I am asking if we can change it to _a and _b instead of _A and _B, which seems awkward inside a DB.